flink-core/src/main/java/org/apache/flink/configuration/SecurityOptions.java

```diff
@@ -132,6 +132,22 @@ public class SecurityOptions {
                     .withDescription(
                             "The time period when keytab login happens automatically in order to always have a valid TGT.");
 
+    @Documentation.Section(Documentation.Sections.SECURITY_AUTH_KERBEROS)
+    public static final ConfigOption<Duration> KERBEROS_TOKENS_RETRY_BACKOFF =
+            key("security.kerberos.tokens.retry.backoff")
```

Comment:
sorry just to connect the 2 configs maybe even better would be `security.kerberos.tokens.renewal.retry.backoff` so they have a common prefix

Reply:
Changed.